modules/ROOT/pages/activity-tracking-edit.adoc

```diff
@@ -0,0 +1,58 @@
+= Activity Tracking
+
+B2B transactions often communicate contractual obligations between companies. Due to the sensitivity of this information, it is desirable (and often necessary) to track transmissions and store both transmission content and metadata.
+
+Anypoint Partner Manager enables you to monitor trading partner transmission activity. Activity is the complete set of end-to-end events related to transmission metadata, messages, and errors that are involved in processing a transmission.
+
+Metadata includes things like the date and time the transmission was sent or received, the identity of the partner, and the success or failure of processing.
+
+Common uses for this metadata include:
+
+* Nonrepudiation
+* Auditing
+* Correlation of acknowledgments
+* Error reporting
+
+With activity tracking, you can:
```

Comment:
I wonder if this text should be earlier in the intro? It seems a little after-the-fact to list uses of tracking after explaining all about metadata, which is part of the tracking but not all of it.

Also...I love the stuff about metadata; just not sure it belongs in this intro to tracking in general? We spend no time here on original payload or errors and acks. Should metadata have its own subtopic?

Reply:
BTW...omg this is so much better! I can actually understand what is going on!!!!